mapie/tests/test_conformity_scores_sets.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+import pytest
+
+# from mapie._typing import ArrayLike, NDArray
+from mapie.conformity_scores import BaseClassificationScore
+from mapie.conformity_scores.sets import APS, LAC, TopK
+from mapie.conformity_scores.utils import check_classification_conformity_score
+
+
+cs_list = [None, LAC(), APS(), TopK()]
+method_list = [None, 'naive', 'aps', 'raps', 'lac', 'top_k']
```

Comment:
Should we add wrong methods here? (and a test to go with it)

Reply:
Done 👍